<?xml version="1.0" encoding="utf-8"?>

<launch>
  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam">
    <remap from="/usb_cam/image_raw" to="/camera/image_raw"/>
  </node>
  <node pkg="ros_opencv_try" type="opencv_subscriber" name="opencv_subscriber"/>
  <node pkg="joy" type="joy_node" name="joy_node"/>
  <node pkg="autojoy" type="joyStickControl" name="joyStick"/>
  <node pkg="autojoy" type="MotorControl" name="motorControl"/>
</launch>
